app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/SampleViewerApplication.kt

```diff
@@ -0,0 +1,16 @@
+package com.esri.arcgismaps.kotlin.sampleviewer
+
+import android.app.Application
+import com.esri.arcgismaps.kotlin.sampleviewer.model.DefaultSampleInfoRepository
+import kotlinx.coroutines.runBlocking
+
+class SampleViewerApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        runBlocking {
```

Comment:
this will block the main thread at app launch. Better to launch this, as you had it before.